README.md

```diff
@@ -18,7 +18,8 @@ the following lines to `~/.bashrc` (or equivalent):
 
 ```bash
 export RAILS_ENV=production
-export DATABASE_ADAPTER=mysql2
+export DATABASE_ADAPTER=sqlite3
```

Comment:
This example now contains parameters that are not compatible. Eg, an sqlite3 database doesn't need a username or password. Would it be better to have a consistent example and then a list of possible variables?